packages/core/src/i18n/index.ts

```diff
@@ -0,0 +1,3 @@
+export * as locales from "./locales/index.js";
```

Comment:
do we need both these first 2 lines? (if so, why exactly)?

Reply:
If anything the first one is the redundant one.

Allows you to import locales either as:
`import { locales } from '@blocknote/core/locales';` or `import { en, nl, de } from '@blocknote/core/locales';`

The `locales` one is just convenient but, can be done with:
`import * as locales from '@blocknote/core/locales';`

So, I can remove it.